TVUIComponent/lib/src/main/AndroidManifest.xml

```diff
@@ -30,55 +28,57 @@ permissions and limitations under the License.
      limitations under the License.
 -->
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.amazon.android.tv.tenfoot"
-          android:versionCode="1"
-          android:versionName="1.1">
+    package="com.amazon.android.tv.tenfoot"
+    android:versionCode="1"
+    android:versionName="1.1">
 
     <uses-sdk
-            android:minSdkVersion="21"
```

Comment:
Could we avoid all those code formatting changes? They will complicate update to new Fire App Builder release in future.